mysql-test/suite/innodb/t/partition_fmt_inconsistency.test

```diff
@@ -0,0 +1,45 @@
+#
+# Test the case when server is aborted after updating InnoDB dictionary,
+# but before fmt file gets updated. This leads to the inconsistency which
+# cannot be auto healied. Informative error message should be printed
+# and there should be possible to drop such a table.
+#
+--source include/have_innodb.inc
+
+--disable_query_log
+call mtr.add_suppression('InnoDB: Partition .* contains 2 user defined columns in InnoDB, but 1 columns in MySQL');
+--enable_query_log
+
+--let $_server_id = `SELECT @@server_id`
+--let $_expect_file_name = $MYSQLTEST_VARDIR/tmp/mysqld.$_server_id.expect
```

Comment:
with expect_crash.inc, we dont need this I think

Reply:
done